packages/map-balloon/src/styledComponents/Balloon.js

```diff
@@ -25,9 +25,10 @@ const open = keyframes`
   }
 `;
 
+const animation = css`animation: ${open} 0.3s ease-in-out;`;
+
 const Balloon = styled(Card)`
-  animation: ${open} 0.3s ease-in-out;
-  position: absolute;
+  ${props => props.animated && animation} position: absolute;
```

Comment:
please add a `;` after animation `${props => props.animated && animation};`
so prettier brakes the line